crypto/merkle/proof_value.go

```diff
@@ -85,8 +85,8 @@ func (op ValueOp) Run(args [][]byte) ([][]byte, error) {
 
 	bz := new(bytes.Buffer)
 	// Wrap <op.Key, vhash> to hash the KVPair.
-	encodeByteSlice(bz, op.key) // nolint: errcheck // does not error
-	encodeByteSlice(bz, vhash)  // nolint: errcheck // does not error
+	encodeByteSlice(bz, op.key) //nolint: errcheck // does not error
+	encodeByteSlice(bz, vhash)  //nolint: errcheck // does not error
```

Comment:
```suggestion
	encodeByteSlice(bz, op.key) 
	encodeByteSlice(bz, vhash) 
```

If these don't error, then there's no reason that errcheck should be upset with us, so the `nolint` direct is unhelpful.